sktime/transformations/series/window_summarizer.py

```diff
@@ -1,434 +1,15 @@
 #!/usr/bin/env python3 -u
 # -*- coding: utf-8 -*-
 # copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
-"""Extract features across (shifted) windows from provided series."""
+"""Implement transformers for summarizing a time series."""
 
-__author__ = ["danbartl"]
-__all__ = ["WindowSummarizer"]
+import warnings
 
-import pandas as pd
-from joblib import Parallel, delayed
+from sktime.transformations.series.summarize import WindowSummarizer
 
-from sktime.transformations.base import BaseTransformer
+warnings.warn(
+    "WindowSummarizer will be moved to summarize. In future, use"
```

Comment:
ok, almost done!
Every deprecation message should include: the time of deprecation, and the time of removal. Here´s the message I would use:

`WindowSummarizer` has been moved to `transformations.series.summarize`, the old location in `series.windows_summarize` is deprecated since 0.11.0, and will be removed in 0.12.0. Please use the import `from transformations.series.summarize import WindowSummarizer`.